src/areaSearch/helpers.ts

```diff
@@ -347,3 +347,8 @@ export const generateAttachmentLink = (
     attachment.id
   }/download`;
 };
+
+export const SPECS_PAGE_INDEX = 0;
```

Comment:
How about somethin along the lines of:
```
export enum AreaSearchStepperPageIndex {
  SPECS = 0,
  APPLICATION = 1,
  PREVIEW = 2,
  SUCCESS = 3
}
```
And then usage would be to import the enum, and replace e.g. `SPECS_PAGE_INDEX` -> `AreaSearchStepperPageIndex.SPECS`